$NetBSD: patch-av,v 1.5 2014/08/14 10:33:29 richard Exp $
Chunk #1:
include both stdint.h and inttypes.h if they are available.
This fixes build failure on Interix-6.1 (sizeof (intmax_t) ==
--- configure.in.orig	2005-10-03 13:34:45.000000000 +0000
+++ configure.in
@@ -394,11 +394,10 @@ else
 		AC_CHECK_SIZEOF(intmax_t, [], [[#include <stdio.h>
 #ifdef HAVE_INTTYPES_H
 #include <inttypes.h>
-#else
+#endif
 #ifdef HAVE_STDINT_H
 #include <stdint.h>
 #endif
-#endif
 ]])
 		AC_CACHE_CHECK(for uniquely sized intmax_t,
 			ccvs_cv_unique_int_type_intmax_t,
@@ -587,7 +586,7 @@ if test -n "$krb_h"; then
            [LDFLAGS=$hold_ldflags
             # Using open here instead of printf so we don't
             # get confused by the cached value for printf from above.
-            AC_CHECK_LIB(krb,open,[krb_lib=yes krb_libdir=])])
+            AC_CHECK_LIB(krb,open,[krb_lib=yes krb_libdir=])],[-ldes -lcom_err -lroken -lcrypt])
        LDFLAGS=$hold_ldflags
   else
        AC_CHECK_LIB(krb,printf,[krb_lib=yes krb_libdir=])
